insights_engine.py

```
@@ -0,0 +1,192 @@
+        year = record.get('year', latest_year)
+        
+        # Determinar tipo de insight
+        emoji = _get_trend_emoji(yoy)
+        trend_text = "crecieron" if yoy > 0 else "decrecieron"
+        
+        # Generar recomendaciones específicas
+        if yoy > 15:
+            action = f"Intensificar promoción comercial y expandir capacidad productiva. Meta: +{yoy*.1:.0f}% adicional en Q4."
+            responsible = "DGCE + MINCETUR"
+        elif yoy > 5:
+            action = f"Consolidar tendencia positiva con misiones comerciales focalizadas."
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)